onnxruntime/contrib_ops/cpu/transformers/subgraph_t5_decoder.cc

```diff
@@ -150,6 +150,8 @@ Status T5DecoderSubgraph::CreateInitialFeeds(
   OrtValue input_ids;
   Tensor::InitOrtValue(DataTypeImpl::GetType<int32_t>(), input_ids_shape, allocator, input_ids);
   int32_t* input_ids_data = input_ids.GetMutable<Tensor>()->MutableData<int32_t>();
+  auto seq_copy = std::make_unique<int32_t[]>(cur_len * batch_beam_size * sizeof(int));
```

Comment:
how about allocating the seq_copy on pinned memory using ORT allocator? https://github.com/microsoft/onnxruntime/blob/main/onnxruntime/contrib_ops/cuda/transformers/generation_device_helper.cc#L199-L202

Reply:
is this the correct way
```
  AllocatorPtr buffer_allocator = std::make_shared<onnxruntime::CPUAllocator>();
  auto seq_copy = IAllocator::MakeUniquePtr<int32_t>(buffer_allocator, cur_len * batch_beam_size * sizeof(int));
  int32_t* seq_copy_ptr = seq_copy.get();
```

Reply:
it's ok to use non-arena based allocator for now